templates/AddSameRegionBucketDownloadRestriction.yaml

```diff
@@ -27,10 +27,12 @@ Parameters:
     AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
     ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
     Default: 'it@sagebase.org'
+  # This lambda, by default, restricts access to buckets when Gateway VPC Endpoints are enabled on an AWS account
+  # This parameter overrides that by explicitly granting bucket access to resources in this list of VPCs.
   SameRegionResourceAccessToBucketGrantVpcAccess:
     Type: CommaDelimitedList
-    Description: Grant bucket access to VPC endpoints (i.e. vpc-111bbb22, vpc-123add45 ). This will allow access via the listed VPCs if SameRegionResourceAccessToBucket is set
-    ConstraintDescription: Comma separated VPC IDs (i.e. vpc-111bbb22, vpc-123add45 )
+    Description: Allow resources in a VPC (i.e. ["vpc-111bbb22, vpc-123add45"]) to access the bucket
```

Comment:
adding `["` and `"]` in the list actually makes a comma delimited list of `"[\"vpc-111bbb22"` and `"vpc-123add45\"]"`. Since I use the python json library to translate a list to json, the end result becomes: `["[\"vpc-111bbb22", "vpc-123add45\"]" ]`

I could make the Lambda take `["vpc-111bbb22", "vpc-123add45"]` as a single string value instead of comma delimited list?

Reply:
I think it's just incorrect documentation.   does `["vpc-111bbb22", "vpc-123add45"]` work? because this is how we've been passing it in for GrantAccess.. https://github.com/Sage-Bionetworks/sandbox-provisioner/blob/master/config/prod/JKG-s3.yaml#L16

which is equivalent to
```  
GrantAccess: ["arn:aws:iam::325565585839:root", "arn:aws:iam::563295687221:user/jake.gockley@sagebase.org"]
```

Reply:
I think it's just incorrect documentation.   does `["vpc-111bbb22", "vpc-123add45"]` work? because this is how we've been passing it in for GrantAccess.. https://github.com/Sage-Bionetworks/sandbox-provisioner/blob/master/config/prod/JKG-s3.yaml#L16

which is equivalent to
```  
GrantAccess: ["arn:aws:iam::325565585839:root", "arn:aws:iam::563295687221:user/jake.gockley@sagebase.org"]
```